<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallHistoryDialog</class>
 <widget class="QDialog" name="CallHistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OutCALL - Call History</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/outcall-logo.png</normaloff>:/images/outcall-logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBoxCalls">
     <property name="title">
      <string> Calls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Missed</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTreeWidget" name="treeWidgetMissed">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <column>
             <property name="text">
              <string>Caller Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Caller Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">To</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Date &amp; Time</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Received</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QTreeWidget" name="treeWidgetReceived">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <column>
             <property name="text">
              <string>Caller Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Caller Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">To</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Date &amp; Time</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Placed</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QTreeWidget" name="treeWidgetPlaced">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>3</number>
            </property>
            <column>
             <property name="text">
              <string notr="true">From</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">To</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Date &amp; Time</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="callButton">
          <property name="text">
           <string>Call</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove selection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addContactButton">
          <property name="text">
           <string>Add Outlok Contact</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
